news/h5p_elink_acc_flags.rst

```diff
@@ -0,0 +1,30 @@
+New features
+------------
+
+* Low-level ``h5p.set_elink_acc_flags`` function that allows to set the external link traversal file access flag in a link access property list.
+* Low-level ``h5p.get_elink_acc_flags`` function that retrieves the external link traversal file access flag from the specified link access property list.
```

Comment:
This is the sort of addition that would generally go under 'Exposing HDF5 functions' - we're exposing additional functions from the HDF5 C API to be called from Python. :slightly_smiling_face:

Reply:
changed news accordingly